dbms/src/Storages/KVStore/Decode/RegionTable.h

```diff
@@ -78,11 +78,15 @@ class RegionTable : private boost::noncopyable
             , range_in_table(range_in_table_)
         {}
 
+        void updateRegionCacheBytes(size_t);
+
         RegionID region_id;
         std::pair<DecodedTiKVKeyPtr, DecodedTiKVKeyPtr> range_in_table;
         bool pause_flush = false;
```

Comment:
Let `pause_flush` be the last member to save memory.

Reply:
Good idea!